### ISSUE TITLE
Synth can't load

### ISSUE THREAD

**Issue by M3198** [mush42/sonata-nvda#42]
Hi,

Hope you're doing well.
Once I download the voices, I want to use them. However, whenever I select sanata tts from the NVDA settings, it gives me the following error: Error:Cannot load speach synthesizer.
Please fix as urgently as possible.

**Comment by mush42** [comment]
Hi
I want more details please.
Can you copy the entire NVDA log after trying to load the synthesizer?

**Comment by mikebayus** [comment]
What version of NVDA are you running?  If you are running an older version, it is very likely that the new version of the add-on won't work as it now works with NVDA 2024.1  I have NVDA 2024.1 running and it works great for me.  Try Piper V2.0-beta2 and see if it works with NVDA 2023.3.4 or earlier.  It should.  Update NVDA.

**Comment by mikebayus** [comment]
Whoa!

You are getting a lot of errors.

Before you updated to the latest NVDA bild did you have a version of Piper?

If so I am given to understand that you can't use the old version of the
voices that you had installed.

I am not a  programer, but might I suggest that you launch the new Sonata
Voice manager while running eloquence and have eloquence help you pick the
voices you want download them and then start NVDA again?

I uninstalled my Piper V20.1-beta2 add-on, installed the new Sonata one,
and I used ESpeak to help me pick the voices I wanted, downloaded them,
restarted NVDA and launched the Sonata synth and I was good to go.

Perhaps you might need to do a clean uninstall of NVDA and reinstall.

As I say, I am not a programer, but that is what I did.

Good luck!

**Comment by M3198** [comment]
Tried all of that. After I deleted the piper folder and things, I created a portable copy without the Sanata addon as well as the associated files with it then I uninstalled NVDA and reinstalled from a portable copy.
Then I tried installing sanata again, still same problem.

**Comment by mikebayus** [comment]
Hmmm!

Works great for me.

I wonder if you actually need windows 11?

Oh yeah, Python is different.

**Comment by M3198** [comment]
I don't think it's a windows 10/11 issue.
When I first installed it it was working. Then when I switched back to eloquence it stopped working, and I've been having a problem since then.
Tried even trying to start the tts using other synths like acapela, same problem.

Let's hope the developer can issue a fix

**Comment by mikebayus** [comment]
Sounds to me like it's an NVDA issue.

I also have Acapela v1.9.2 and it also works for me no problem.

I wonder if it's your eloquence Add-on.

What happens if you disable it, or remove it, then uninstall and reinstall
NVDA?

Or take everything out and do a cleaninstall of NVDA but not with the
eloquence add-on.

I once had a similar problem with Cepstral Amy and Acapela and the Ella
voice.

Amy and Ella simply did not like each other.

Amy was a SAPI5 voice, and all of my other SAPI5 voices worked but not Amy
unless I disabled my Acapela for NVDA Add-on.

I told Cepstral about it, but they had no idea what I was talking about, so
they did nothing.

Not only that, but Acapela was the other guy.

I assume that your eloquence is from Code Factory.

And they offer no support.

If you want Sonata, you may just have to give up eloquence.

I hope that won't be the case.

Good luck my friend.

**Comment by mush42** [comment]
Weird.
Another update is coming. Maybe it'll fix this issue.

**Comment by M3198** [comment]
Tried that, still same issue.
I don't think it's to do with eloquence.

**Comment by M3198** [comment]
Just to ask, what's the easiest way to reply on github?
When I receive a notification that my issue has been replied to, should I directly reply to the email normally i.e. pressing control r in thunderbird for example?
I'm new to this github, by the way.
Best regards,
Muhammad.

**Comment by mush42** [comment]
@M3198  please try the latest release.
